lib/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/validation/DcmawEvidenceValidator.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.di.ipv.core.library.domain.gpg45.validation;
```

Comment:
I think this class is identical to `PassportEvidenceValidator`.
When I created these I should probably have written them with the GPG45 elements in mind, rather than the CRIs. They could probably be abstracted to just refer to GPG45 rather than CRIs.

Reply:
Ah yeah true. I'll remove this one and rename them to refelect GPG45 types rather than CRI's